src/components/Company/CompanyName/CorrectNameOptions.vue

```diff
@@ -74,16 +97,22 @@ export default class CorrectNameOptions extends Vue {
     if (this.correctionNameChoices.length === 1) this.panel = 0 // open by default if only 1 option
   }
 
-  /** save name correction */
+  /** Request the child to submit it's form */
```

Comment:
I don't understand this comment. It looks like the child is asking the parent to submit?

Reply:
Sorry, poor phrasing. I will update this :)